scss/_patterns_navigation.scss

```diff
@@ -100,31 +127,37 @@
       }
 
       .p-navigation__link {
```

Comment:
@anthonydillon Ah okay, well maybe I'll make an issue for the external link and to get an appropriate asset for it.